apps/a11y-tests/src/axe-sarif-converter/axe-utils.ts

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+/// <reference path="./axe-extension.ts" />
+import * as axe from 'axe-core';
+
+import { DictionaryStringTo } from './dictionary-types';
+
+export type ImageCodedAs = 'Decorative' | 'Meaningful';
+
+export function getMatchesFromRule(ruleId: string): (node: any, virtualNode: any) => boolean {
```

Comment:
Can you figure out a better type besides `any` here? (same comment anywhere else you use `any`)